src/main.rs

```diff
@@ -19,5 +19,11 @@ fn main() {
 
     // spawn auditor thread from block store to WSV
     //TODO:
-    networking::torii::Torii::new().start();
+    networking::torii::Torii::new(consensus::sumeragi::Sumeragi::new(
+        model::block::Blockchain::new(
+            async_std::task::block_on(storage::kura::Kura::strict_init())
```

Comment:
We can use `futures::block_on` now. Personally I think it's better to use async_std only for working with std IO and futures for executors, now that we have them. To make code more readable.

Reply:
done